### ISSUE TITLE
NVDA should have a skip to non-link text function

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#165]
**Reported by mike reiser on 4 May 2008 03:17 UTC**
NVDA should have a key that allows one to skip to non-linked text on the page.  Eather that or it could jump right to the text in question, similar to how system access does this.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 4 May 2008 03:34 UTC**
Duplicate of #71.
